Earthfile

```diff
@@ -36,6 +36,13 @@ test-install-dind-arm64:
 test-install-dind-for-image:
     ARG base_image
     FROM "$base_image"
+
+    # docker-compose will fail with:
+    #     Error response from daemon: Invalid container name (-hello-1), only [a-zA-Z0-9][a-zA-Z0-9_.-] are allowed
+    # if docker-compose.yml is in the root directory, due to the --project flag defaulting to "".
+    # This bug does not exist in the official earthly/dind:alpine image.
```

Comment:
https://github.com/linuxserver/docker-docker-compose has the 2.X releases of docker-compose marked as pre-release; once we get 2.X, I suspect this bug will be fixed and we could remove the `WORKDIR /dind-test` work-around.